editor/src/clj/editor/keymap.clj

```diff
@@ -36,6 +36,7 @@
            ["Alt+Down" :end-of-line]
            ["Alt+Down" :move-down]
            ["Alt+Left" :prev-word]
+           ["Alt+F9" :edit-breakpoint]
```

Comment:
I think we should add a menu item for this, since we already have one for Toggle Breakpoint. This would help users discover the added keyboard shortcut.